### ISSUE TITLE
🐛[BUG][NPC/Quest/Mission/System/Combat/Launcher/Website]

### ISSUE THREAD

**Issue by Xenith-the-Red** [HorizonFFXI/HorizonXI-Issues#1243]
NM "Diamond Daig" doesn't seem to pop here.

It should be lottery off two beetles (I-7 and G-9) and pops in the central hall at H-8, during 2x Sandstorm only.

I've tried to get it to pop for about 2 weeks.  Keeping my character in Quicksand for past week.  It has never taken this many kills;  it was a generous lottery on era retail (as well as other servers).

**Comment by CriticalXI** [comment]
Unfortunately the code seems correct and this NM does indeed spawn. Seems to be really bad RNG for you. :\

**Comment by Reeno23** [comment]
I got my bangles today (1/9/24) and can confirm Daig can pop. However, the placeholder is wrong. The SW, NE, far east middle, and far west middle beetles are all 7 min respawns and not related to Daig. The placeholder ended up being the 'random' middle of hallway beetle that has a 16 min respawn. I was tracking timers on every spawn all day and he spawned when none of the others were close to spawning. It also spawned right where the middle beetle spawns. I did only do testing with double earth, so I'm sure that requirement is in place if @CriticalXI saw this in the code. No biggie with the 16min beetle being the PH, just tough without knowing or if you're content killing every beetle in sight anyway.
